pkg/controller/common/util.go

```diff
@@ -320,6 +320,9 @@ const (
 	AnnEventSourceKind = "cdi.kubevirt.io/events.source.kind"
 	// AnnEventSource is the source that should be related to events (namespace/name)
 	AnnEventSource = "cdi.kubevirt.io/events.source"
+
+	// AnnAllowClaimAdoption is the annotation that allows a claim to be adopted by a DataVolume
+	AnnAllowClaimAdoption = "cdi.kubevirt.io/allowClaimAdoption"
```

Comment:
Do we like this value?  `allowClaimAdoption` does not use dotted naming like others?

Reply:
I potentially like "allow" as a value but still not sure about naming.  Any other ideas?  ss @akalenyu